kernel/kernel-impl/src/main/java/org/sakaiproject/event/impl/BaseEventTrackingService.java

```diff
@@ -225,10 +226,31 @@ public Event newEvent(String event, String resource, boolean modify, int priorit
 	 */
 	public Event newEvent(String event, String resource, String context, boolean modify, int priority)
 	{
-		return new BaseEvent(event, resource, context, modify, priority);
+		return new BaseEvent(event, resource, context, modify, priority, null);
 	}
 
 	/**
+	 * Construct a Event object.
+	 *
+	 * @param event
+	 *        The Event id.
+	 * @param resource
+	 *        The resource reference.
+	 * @param context
+	 *        The Event's context.
+	 * @param modify
+	 *        Set to true if this event caused a resource modification, false if it was just an access.
+	 * @param priority
+	 *        The Event's notification priority.
+	 * @return A new Event object that can be used with this service.
+	 */
+	public Event newEvent(String event, String resource, String context, boolean modify, int priority, LRS_Statement lrsStatement)
+	{
```

Comment:
Yeah, the whole file is formatted like this though. While I hate this brace style, I think it's better to just stick with what's there. I *usually* try to stay with whatever format as much as I don't like it is in the original code. In SimpleEvent it was already formatted different (and more correct) in the whole file.